src/gui/statsdialog.cpp

```diff
@@ -77,7 +77,10 @@ void StatsDialog::update()
                 : "-");
     // Cache hits
     qreal readRatio = cs.readRatio;
-    m_ui->labelCacheHits->setText((readRatio >= 0) ? Utils::String::fromDouble(100 * readRatio, 2) : "-");
+    m_ui->labelCacheHits->setText(QString("%1%")
+                                    .arg(readRatio > 0
+                                        ? Utils::String::fromDouble(100 * readRatio, 2)
+                                        : "0"));
```

Comment:
IMO, weird indentation: many lines, too offset to the right.
Maybe the following:
```c++
m_ui->labelCacheHits->setText(
                QString("%1%").arg(readRatio > 0 
                                   ? Utils::String::fromDouble(100 * readRatio, 2) : "0"));
```
or even:
```c++
m_ui->labelCacheHits->setText(
                (readRatio > 0)
                ? QString("%1%").arg(Utils::String::fromDouble(100 * readRatio, 2)) 
                : "0%");
```